.github/workflows/bindings-java.yml

```diff
@@ -54,7 +48,7 @@ jobs:
           cache: true
           cache-root: client/bindings/java/lib/native/
           cache-job-id: ${{ github.workflow }}-${{ github.job }}-${{ matrix.os }}-${{ matrix.java }}
```

Comment:
Does it make sense to remove the language specific caches now? `-${{ matrix.java }}`

Reply:
Yeah maybe

Reply:
But if you allow me I'll do a dedicated PR for caching, there are too many things to explore and I need to chat with Rajiv.